src/main/java/survey/model/Survey.java

```diff
@@ -67,6 +67,8 @@ public void setQuestions(List<Question> questions) {
 
     public void addQuestion(Question question) { this.questions.add(question); }
 
+    public boolean getOpen(){ return this.open; }
```

Comment:
When would we need this method?

Reply:
we need it to aactually send if the survey is open or closed, otherwise when we send the survey it doesnt send this